apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Mutations.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Protobuf;
+using System;
+
+namespace Google.Cloud.Bigtable.V2
+{
+    /// <summary>
+    /// Static class with convenience methods for creating various kinds of <see cref="Mutation"/> instances.
+    /// </summary>
+    public static class Mutations
+    {
+        /// <summary>
+        /// Creates a <see cref="MutateRowsRequest.Types.Entry"/> to use with MutateRows requests.
```

Comment:
So what is an Entry here? It may be worth repeating some of the autogenerated docs just for clarity.

Reply:
Unfortunately, the `Entry` type itself has no docs. The param docs are copied though from its auto-generated properties. I'll add some more to the summary and returns to clarify.